src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -527,6 +528,65 @@ public void WriteByteArray(byte[] bytes)
             Write(bytes);
         }
 
+        public async Task<bool> WriteConst(ConstantTypeCode? type, byte[] bytes, MonoSDBHelper SdbHelper, CancellationToken token)
```

Comment:
Would it make sense to move the Default extraction logic into ParameterInfo and avoid having the binary writer understand the details of the typecode and byte packing?